docs/use_a_template.md

```diff
@@ -21,11 +21,15 @@ If you don't want to download the manifest file locally, you can point to its re
 If you are using InfluxDB Cloud platform you will need provide the URL of your InfluxDB zone using the `--host` flag. For example `https://us-west-2-1.aws.cloud2.influxdata.com`. 
 
 ```
- influx pkg --org <organization_name> --file ~/path/to/template/manifest.yml --host <az_name> --token <token>
+ influx pkg --org <organization_name> --file ~/path/to/template/manifest.yml --host <influxdb_cloud_host> --token <token>
 ```
-You can find your tokens or create a new one under the `Load Data` -> `Tokens` section of the UI.
+
+> Your default organizational name in InfluxDB Cloud will be your email address.
+
+In order to apply Templates and download Telegraf configurations from InfluxDB Cloud you will need to create an **All Access Token** for use on the command line. You can create this token from the `Load Data` -> `Tokens` section of the UI.
```

Comment:
```
To apply templates and download Telegraf configurations from InfluxDB Cloud, create an **All Access Token**. Create this token from the `Load Data` -> `Tokens` section of the UI.
```

You could also included this link to the docs for creating a token: https://v2.docs.influxdata.com/v2.0/security/tokens/create-token/